Compiled from "TestProcedureFuture.java"
class org.apache.hadoop.hbase.client.TestProcedureFuture$2 extends org.apache.hadoop.hbase.client.TestProcedureFuture$TestFuture {
  final org.apache.hadoop.hbase.client.TestProcedureFuture this$0;

  org.apache.hadoop.hbase.client.TestProcedureFuture$2(org.apache.hadoop.hbase.client.TestProcedureFuture, org.apache.hadoop.hbase.client.HBaseAdmin, java.lang.Long);
    Code:
       0: aload_0
       1: aload_1
       2: putfield      #1                  // Field this$0:Lorg/apache/hadoop/hbase/client/TestProcedureFuture;
       5: aload_0
       6: aload_2
       7: aload_3
       8: invokespecial #2                  // Method org/apache/hadoop/hbase/client/TestProcedureFuture$TestFuture."<init>":(Lorg/apache/hadoop/hbase/client/HBaseAdmin;Ljava/lang/Long;)V
      11: return

  protected org.apache.hadoop.hbase.shaded.protobuf.generated.MasterProtos$GetProcedureResultResponse getProcedureResult(org.apache.hadoop.hbase.shaded.protobuf.generated.MasterProtos$GetProcedureResultRequest) throws java.io.IOException;
    Code:
       0: aload_0
       1: aload_1
       2: invokespecial #3                  // Method org/apache/hadoop/hbase/client/TestProcedureFuture$TestFuture.getProcedureResult:(Lorg/apache/hadoop/hbase/shaded/protobuf/generated/MasterProtos$GetProcedureResultRequest;)Lorg/apache/hadoop/hbase/shaded/protobuf/generated/MasterProtos$GetProcedureResultResponse;
       5: pop
       6: new           #4                  // class org/apache/hadoop/hbase/DoNotRetryIOException
       9: dup
      10: new           #5                  // class java/lang/UnsupportedOperationException
      13: dup
      14: ldc           #6                  // String getProcedureResult
      16: invokespecial #7                  // Method java/lang/UnsupportedOperationException."<init>":(Ljava/lang/String;)V
      19: invokespecial #8                  // Method org/apache/hadoop/hbase/DoNotRetryIOException."<init>":(Ljava/lang/Throwable;)V
      22: athrow
}
